api/pkg/db/migration/migration.go

```diff
@@ -55,9 +56,6 @@ func Migrate(api *app.APIBase) error {
 			&model.SyncJob{},
 			&model.Scope{},
 			&model.Config{},
-			&model.Platform{},
```

Comment:
Why are we removing these from here ?

Reply:
These were added  in the migration as well

Reply:
why are removing `platform`? the other 2 will be autocreated how platform will get created on a new db installation if we remove from here?

Reply:
here platforms is optional and we can keep or remove it,if we remove platforms from here it will get created by the automigration on a new db installation

Reply:
okay but I didn't understood why this is optional? 🤔 on new db creation `initschema` https://github.com/tektoncd/hub/blob/3c1ec7d356748afce2281879662989d5d8fb3d10/api/pkg/db/migration/migration.go#L47  is run right?
and if platforms is not added there still it will get created

Reply:
yes

Reply:
So platform table is getting created because of this migration https://github.com/tektoncd/hub/blob/cc843047e14327e18a8d12de66083cbcb3b5145b/api/pkg/db/migration/202106281500_create_platform_tables.go#L24